### ISSUE TITLE
EmailDeciderTest failure

### ISSUE THREAD

**Issue by schneems** [codetriage/codetriage#541]
```
EmailDeciderTest#test_decides_email_frequency [/home/travis/build/codetriage/codetriage/test/unit/email_decider_test.rb:16]:
Expected  EmailDecider.new(3, 'once_a_week').now?(238) to be false, was not
```

**Comment by nateberkopec** [comment]
i've seen this randomly too, pretty weird

**Comment by nateberkopec** [comment]
This must be a test ordering thing.

**Comment by nateberkopec** [comment]
Or maybe a legit failure from the random days_ago

**Comment by schneems** [comment]
Or a time zone thing. All the values from those random ranges should be valid.

**Comment by lostapathy** [comment]
Note: this test got renamed to EmailRateLimitTest.

Are you guys still seeing this, or has it gone away?  Do you need to run the full test suite to get it to happen, or will it happen just running that test?

**Comment by lostapathy** [comment]
@nateberkopec  I believe this can be closed now that #633 has been merged.  I guess I should have noted that on the PR so this would have auto-closed.
